public/app/factories/versionbumper.factory.js

```diff
@@ -10,11 +10,45 @@ angular
 			$http,
 			configuration
 		) {
-			var api = configuration.serverPath + configuration.apiPrefix + configuration.apiLevel;
+			var api = configuration.serverPath + configuration.apiPrefix + configuration.apiLevel + "acpaasportalversionbumper";
 			var factory = {};
 
+			function handleResponse(response) {
```

Comment:
Binnen het WCM maken we meestal gebruik van $resource maar dit is ook goed voor mij.